packages/server/api/routes/favorites.router.js

```diff
@@ -27,10 +27,10 @@ const favoritesController = require('../controllers/favorites.controller');
  *        description: Unexpected error.
  */
 router.get('/', (req, res, next) => {
-  const userId = 1;
-  // TO DO : once we will add authentication I will update it
+  const { token } = req.headers;
```

Comment:
I don't really understand why we are passing the token here since we don't actually use it to verify that the user is authorized for this request. We could just unpack the user ID in the UserContext and send that instead. 

Or at least unpack the userID in the router so that the controller doesn't have to know about that step.